src/style-spec/reference/v8.json

```diff
@@ -181,6 +181,19 @@
       "type": "promoteId",
       "doc": "A property to use as a feature id (for feature state). Either a property name, or an object of the form `{<sourceLayer>: <propertyName>}`. If specified as a string for a vector tile source, the same property is used across all its source layers."
     },
+    "volatile": {
+      "type": "boolean",
+      "default": false,
+      "doc": "A setting to determine whether a source's tiles are cached locally.",
+      "sdk-support": {
+        "basic functionality": {
+          "js": "Not yet available",
```

Comment:
Oh, doesn't the "not yet available" show up automatically if you exclude the platform from the `basic functionality: { }` section?